tikvrpc/tikvrpc.go

```diff
@@ -275,6 +275,12 @@ func (req *Request) EnableStaleRead() {
 	req.ReplicaRead = false
 }
 
+// DisableStaleRead is called when stale-read fallbacks to leader read after meeting key-is-locked error.
+func (req *Request) DisableStaleRead() {
```

Comment:
How about `DisableStaleReadAfterFallback`

Reply:
Rename it with `DisableStaleReadMeetLock` because it should only take effect when key is locked, other rollback won't affect this flag